monero/caip10.md

```diff
@@ -0,0 +1,76 @@
+---
+namespace-identifier: monero-caip10
+title: Monero - Addresses
+author: silverpill (@silverpill)
+discussions-to: https://github.com/ChainAgnostic/namespaces/issues/41
+status: Draft
+type: Informational
+created: 2023-09-01
+requires: ["CAIP-10"]
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
+
+## Rationale
+
+CAIP-10 defines a pattern for identifying an account in a blockchain. 
+Monero chains use UTXO model and [Monero addresses][] are functionally similar to accounts in other namespaces.
```

Comment:
```suggestion
Monero chains use the UTXO accounting model, making [Monero addresses][] functionally similar to other UTXO-based namespaces, in that CAIP-10 identifiers represent **payee identifiers** (of which a given actor has many) rather than stable **actor identifiers**.
```

Reply:
>CAIP-10 identifiers represent **payee identifiers** (of which a given actor has many) rather than stable **actor identifiers**

I think this is not correct. Each Monero address represents a separate identity. It has its own balance, it can be re-used, etc.

Reply:
Hmmm, you make a good point, they are independent and stable identities and in that sense a good analogue to an EVM-style account-address; what I'm confused on is the _independent balance_ part, which is where I thought the analogy broke down (to the glee of privacy maxis who ridicule EVM 😅 ).  

As for `payee`, I was kind of leaning on my admittedly superficial understanding of BTC terminology, where P2PK, P2PKH, and P2SH addresses are all "payees" (legitimate targets of a txn) even though they're not all actors or accounts. 

Can you say more about the independent balance? I was under the impression that I send 10 coins each to 10 different subaddresses of an empty wallet, the raw address of that account now has 100 coins it can send on to some third party?  I.e., I thought the "balances" of subaddresses were independent (but partial) views of a single balance (in the underlying ledger/accounting sense),  am I wrong?

Reply:
For practical purposes, subaddresses are distinct "boxes" with separate balances. See this RPC endpoint for example: https://www.getmonero.org/resources/developer-guides/wallet-rpc.html#get_balance

I don't know enough to say how it works under the hood. Generally, addresses in UTXO-based systems are abstractions, and on the base level there are discrete "notes" which you can spend only if you know the secret.